### PR TITLE
Update rate limit error copy for pro users

### LINKED ISSUE #194: bug: Change the copy of the JetBrains Cody Pro user limit copy
### Installation Information

NA

### Goal
To prevent Cody Pro users from abusing Cody Pro by spamming a bunch of chat/command and autocomplete requests.

### Describe the bug

Current copy:
- Everything under the right hand column (aka Pro messages) in this comment: https://github.com/sourcegraph/jetbrains/pull/166#issuecomment-1843517258

### Expected behavior

Requested copy:
- **Thank you for using Cody so heavily today!** 
- To ensure that Cody can stay operational for all Cody users, please come back tomorrow for more chats, commands, and autocompletes. [Learn more](https://docs.sourcegraph.com/cody/core-concepts/cody-gateway#rate-limits-and-quotas)

### Additional context (logs, images, etc)

_No response_

### PR DESCRIPTION
**Pull request by mkondratek**
Fixes https://github.com/sourcegraph/jetbrains/issues/194.

## Test plan
- play with the accounts with rate limit exceeded

### CODE REVIEW

**Comment by mkondratek** [comment]
It is impossible to test it right now b/c of the issue with the agent (PLG Rate limiting error messages are currently broken).

**Comment by mkondratek** [comment]
image

image



Warning dialog from the status widget:

image

**Comment by chenkc805** [comment]
LGTM.

File: src/main/kotlin/com/sourcegraph/cody/chat/Chat.kt [hunk 1]
```diff
@@ -92,9 +92,9 @@ class Chat {
                       "(Already upgraded to Pro? Restart your IDE for changes to take effect)"
                 }
                 else -> {
-                  "<b>Request failed:</b> You've used all${rateLimitError.quotaString()} chat messages and commands." +
```
Comment (pkukielka) [inline src/main/kotlin/com/sourcegraph/cody/chat/Chat.kt:95]: Out of curiosity why we are not using text bundle everywhere but instead copy paste the same message in different places?

Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/chat/Chat.kt:95]: I think we should use the bundle everywhere. I initialized that during the rough time around GA for some specific purpose - the bundle makes things cleaner and more readable imho.

**Review by exigow — APPROVED** [review]

**Pull request merged**